README.md

```diff
@@ -106,44 +112,46 @@ Other shortcuts:
 | Alt+Enter | Fullscreen (DirectX) |
 | Ctrl+R | Reset |
 
-Currently, DirectX 11 and OpenGL are supported. Change the renderer by opening the `shipofharkinian.json` configuration file in notepad and add `sdl` to the quotes in `"GfxBackend": ""` for OpenGL or leave blank for DirectX.
+Currently, DirectX 11 and OpenGL are supported. You can change which API to use in the `Settings` menu of the menubar, which requires a restart.  If you're having an issue with crashing, you can change the API in the `shipofharkinian.json` file by finding the line `gfxbackend:""` and changing the value to `sdl` for OpenGL. DirectX 11 is the default on Windows.
```

Comment:
Maybe worth mentioning that Metal is available on Mac now.

Reply:
I was under the impression that the Metal builds were still limited to nightlife. Has that changed? If so, then yeah, I can add that.

Reply:
Yup Metal became available in 6.1.0 release

Reply:
Got it!